src/js/profile/application_settings.js

```diff
@@ -119,6 +121,18 @@ export const allApplicationSettings = [
     new BoolSetting("offerHints", categoryGame, (app, value) => {}),
 ];
 
+/** @type {Array<BaseSetting>} */
+export const allDebugSettings = [];
+for (const k in globalConfig.debug) {
+    if (!IS_DEBUG) break;
```

Comment:
Better wrap the whole block in IS_DEBUG

Reply:
I chose to disable them instead. Request changes again if appropriate.